core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -167,11 +224,33 @@ export class CreateServiceComponent implements OnInit {
     if(!this.disablePlatform){
       this.typeOfPlatform = platformType;
     }
+    if(document.getElementById('deployment-type')){
+      this.scrollTo('deployment-type');
+    }     
+    else{
+      this.scrollTo('runtime-type');
+    }
+
   }
 
   // function called on runtime change(radio)
   onSelectionChange(val){
     this.runtime = val;
+    this.typeform = true;
```

Comment:
If the user has already entered something in Start New and then changes the runtime, does this handle that scenario by keeping what the user entered rather than loading from the template?

Reply:
Made this change
if start new is selected, the textarea will be empty
else it will load the yaml text of respective runtime